### PR TITLE
Citekey markup

### PR DESCRIPTION
**Pull request by adam3smith**
Fixes https://github.com/zotero/translators/issues/853
Since this is pretty small, I'll squash and accept tomorrow night unless there are any concerns.

### CODE REVIEW

File: BibTeX.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	"creator": "Simon Kornblith, Richard Karnesky and Emiliano heyns",
 	"target": "bib",
 	"minVersion": "2.1.9",
-	"maxVersion": "",
+	"maxVersion": null,
```
Comment (aurimasv) [inline BibTeX.js]: How did this happen?

Comment (adam3smith) [inline BibTeX.js]: this is what my local bibtex.js copy had. We want the empty quotes?

Comment (aurimasv) [inline BibTeX.js]: yes. Thought maybe Scaffold is acting up.

File: BibTeX.js [hunk 2]
```diff
@@ -1050,7 +1050,8 @@ var numberRe = /^[0-9]+/;
 // in includes the indefinite articles of English, German, French and Spanish, as well as a small set of English prepositions whose 
 // force is more grammatical than lexical, i.e. which are likely to strike many as 'insignificant'.
 // The assumption is that most who want a title word in their key would prefer the first word of significance.
-var citeKeyTitleBannedRe = /\b(a|an|the|some|from|on|in|to|of|do|with|der|die|das|ein|eine|einer|eines|einem|einen|un|une|la|le|l\'|el|las|los|al|uno|una|unos|unas|de|des|del|d\')(\s+|\b)/g;
+// Also remove markup
+var citeKeyTitleBannedRe = /\b(a|an|the|some|from|on|in|to|of|do|with|der|die|das|ein|eine|einer|eines|einem|einen|un|une|la|le|l\'|el|las|los|al|uno|una|unos|unas|de|des|del|d\')(\s+|\b)|(\<(i|b|sup|sub|sc|span style=\"small-caps\")\>)|(<\/(i|b|sub|sup|sc|span)\>)/g;
```
Comment (aurimasv) [inline BibTeX.js]: nit: don't need to escape `<` or `>` and you don't need to duplicate all the tags, you can just list them once with `\/?`

**Pull request merged**